mappings/net/minecraft/client/gui/DrawableHelper.mapping

```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/class_681 net/minecraft/client/gui/DrawableHelper
-	FIELD field_2866 BACKGROUND_TEXTURE Lnet/minecraft/class_1605;
+	FIELD field_2866 OPTIONS_BACKGROUND_TEXTURE Lnet/minecraft/class_1605;
```

Comment:
its not just used in the options, its used anywhere a transparent background isnt

Reply:
I don't like the term `BACKGROUND_TEXTURE` because I find it too general.
Also on [newer versions](https://github.com/FabricMC/yarn/blob/eb10920c4fb3ed87e8c41ad956e798db757d869e/mappings/net/minecraft/client/gui/DrawableHelper.mapping#L4) it's also called like that.

Reply:
> its not just used in the options, its used anywhere a transparent background isnt

fake news

Reply:
> I don't like the term `BACKGROUND_TEXTURE` because I find it too general.
> Also on [newer versions](https://github.com/FabricMC/yarn/blob/eb10920c4fb3ed87e8c41ad956e798db757d869e/mappings/net/minecraft/client/gui/DrawableHelper.mapping#L4) it's also called like that.

if its from upstream then keep it

Reply:
@hYdos :b:ad